sklearn/preprocessing/data.py

```diff
@@ -863,7 +863,7 @@ class RobustScaler(BaseEstimator, TransformerMixin):
     ----------
     with_centering : boolean, True by default
         If True, center the data before scaling.
-        This does not work (and will raise an exception) when attempted on
+        This will cause ``transform`` to raise an exception when attempted on
```

Comment:
Yeah, `fit`on sparse raises regardless of any option. You can `transform` on sparse after `fit`ting on dense, apparently?